Flutter/calendar/appointments.md

```diff
@@ -130,6 +130,129 @@ events.notifyListeners(CalendarDataSourceAction.reset, null);
 {% endhighlight %}
 {% endtabs %}
 
+## Datasource manipulation
+
+By call this method whenever the datasource changes, to notify the datasource may have. Need to notify the datasource action(add, remove, and reset) through notify listener. In the following example, the appointment and resource are added, removed, and replaced and notifyListeners is called in onPressed callback of the FloatingActionButton.
+
+#### Add Action
+
+When dynamically adding appointments to a data source, we must notify the data source for the add action.
+
+{% tabs %}
+{% highlight dart hl_lines="14 15 16" %}
```

Comment:
highlight 8 - 16

Reply:
I have checked and highlighted the line from 8 to 16. I have updated my changes in the latest commit.